values.yaml

```diff
@@ -370,7 +410,19 @@ st2web:
   securityContext: {}  # NB: nginx requires some capabilities, drop ALL will cause issues.
   # mount extra volumes on the st2web pod(s) (primarily useful for k8s-provisioned secrets)
   ## Note that Helm templating is supported in 'mount' and 'volume'
-  extra_volumes: []
+  extra_volumes:
+    - name: st2-cert
+      mount:
+        mountPath: "/etc/ssl/st2/"
+        readOnly: true
+      volume:
+        secret:
+          secretName: "stackstorm-tls{{if .Values.phaseCanary}}-canary{{end}}"
```

Comment:
phaseCanary is not defined in our values file.

Reply:
Whoops, that was an internal change I missed removing - fixed it now